### PR TITLE
Add documentation about different installations options

### PR DESCRIPTION
**Pull request by ajfriesen**
**Description:**

A lot of people do not know what the real difference between all this installations methods are.
Then there are Hass.io Add-Ons which get people confused as well.
I was also in the place and it did take long to understand that. (Maybe I am still not there ;-))

This could also go to the FAQ as well, just started a draft.

The graphics are really not fancy but will hopefully make something more clear.
Made with yed. Added the hassio_on_hassos.graphml file for later editing.

What do you think?

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_docs/installation.markdown [hunk 1]
```diff
@@ -131,3 +131,124 @@ For more details about `hass`, please refer to the [tools section](/docs/tools/h
...
+<div class="text-center" markdown="0">
+    <div class='img-container'>
+      <img src='/images/docs/architecture/hassio_on_generic_linux_server.svg' />
+    </div>
+</div>
+
+It is almost the same like Hass.io on HassOS but you do run Hass.io on your own operating system of choice instead of HassOS.
+
+You will get:
+- full Home Assistant
```
Comment (klaasnicolaas) [inline source/_docs/installation.markdown]: What do you mean with this?

**Comment by frenck** [comment]
I'm going to assign myself as reviewer, quick glance, there are quite a bit of incorrect statements in here and spotted some spelling/grammar errors.

Will clarify/review this later (dinner first).

**Comment by DubhAd** [comment]
IMO, this kind of guide would be better done on its own page.

File: source/_docs/installation.markdown [hunk 2]
```diff
@@ -131,3 +131,124 @@ For more details about `hass`, please refer to the [tools section](/docs/tools/h
...
 You may need to install additional libraries depending on the platforms/components you want to use.
+
+## {% linkable_title Characteristics of different install methods for Home Assistant %}
+
+You always get the full Home Assistant expirience despite the installation method you choose to use.
+How you would like to run Home Assistant is a choice of taste, the setup you already have, knowlede and what you want to accomplish with home assistant.
+
+<p class='note'>
+Add-Ons provide an easy way to add more functionality to your setup through the gui.
+Add-Ons in Hass.io are just docker containers containing some software.
```
Comment (frenck) [inline source/_docs/installation.markdown]: Basically, correct, nevertheless, they do have more integrations possibilities in comparison to plain Docker.

```suggestion
Add-ons in Hass.io are essentially just docker containers containing some software.
```

File: source/_docs/installation.markdown [hunk 3]
```diff
@@ -131,3 +131,124 @@ For more details about `hass`, please refer to the [tools section](/docs/tools/h
...
+    <div class='img-container'>
+      <img src='/images/docs/architecture/hassio_on_generic_linux_server.svg' />
+    </div>
+</div>
+
+It is almost the same like Hass.io on HassOS but you do run Hass.io on your own operating system of choice instead of HassOS.
+
+You will get:
+- full Home Assistant
+- Hass.io Add-Ons (some may not work properly)
```
Comment (frenck) [inline source/_docs/installation.markdown]: `(some may not work properly)` this is actually incorrect. Add-ons may have an architecture limitation, so do docker containers or any other piece of software you'd like to install on a certain computer.

File: source/_docs/installation.markdown [hunk 4]
```diff
@@ -131,3 +131,124 @@ For more details about `hass`, please refer to the [tools section](/docs/tools/h
 If you're running a Linux-based platform, we suggest you follow the [VirtualEnv instructions](/docs/installation/virtualenv/) to avoid using `root`.
 
 You may need to install additional libraries depending on the platforms/components you want to use.
+
+## {% linkable_title Characteristics of different install methods for Home Assistant %}
+
+You always get the full Home Assistant expirience despite the installation method you choose to use.
```
Comment (frenck) [inline source/_docs/installation.markdown]:
```suggestion
You always get the full Home Assistant experience despite the installation method you choose to use.
```

File: source/_docs/installation.markdown [hunk 5]
```diff
@@ -131,3 +131,124 @@ For more details about `hass`, please refer to the [tools section](/docs/tools/h
...
+How you would like to run Home Assistant is a choice of taste, the setup you already have, knowlede and what you want to accomplish with home assistant.
```
Comment (frenck) [inline source/_docs/installation.markdown]:
```suggestion
How you would like to run Home Assistant is a choice of taste, the setup you already have, knowledge and what you want to accomplish with Home Assistant.
```

File: source/_docs/installation.markdown [hunk 6]
```diff
@@ -131,3 +131,124 @@ For more details about `hass`, please refer to the [tools section](/docs/tools/h
...
+
+<p class='note'>
+Add-Ons provide an easy way to add more functionality to your setup through the gui.
```
Comment (frenck) [inline source/_docs/installation.markdown]:
```suggestion
Add-ons provide an easy way to add more functionality to your setup through the GUI.
```

File: source/_docs/installation.markdown [hunk 7]
```diff
@@ -131,3 +131,124 @@ For more details about `hass`, please refer to the [tools section](/docs/tools/h
...
+Add-Ons in Hass.io are just docker containers containing some software.
+They are configured to for Home Assistant for easy setup.
+You always can add the functionaliy provided by an Add-On by yourself if you are **not** running Hass.io.
```
Comment (frenck) [inline source/_docs/installation.markdown]:
```suggestion
You always can add the functionality provided by an add-on by yourself if you are **not** running Hass.io.
```

File: source/_docs/installation.markdown [hunk 8]
```diff
@@ -131,3 +131,124 @@ For more details about `hass`, please refer to the [tools section](/docs/tools/h
...
+### Hass.io on HassOS
+
+<div class="text-center" markdown="0">
+    <div class='img-container'>
+      <img src='/images/docs/architecture/hassio_on_hassos.svg' />
+    </div>
+</div>
+
+This is a full appliance for running Home Assistant on your hardware using docker images running on HassOS.
+Home Assistant is running as a docker container as well as Add-Ons.
```
Comment (frenck) [inline source/_docs/installation.markdown]:
```suggestion
Home Assistant is running as a docker container as well as add-ons.
```

File: source/_docs/installation.markdown [hunk 9]
```diff
@@ -131,3 +131,124 @@ For more details about `hass`, please refer to the [tools section](/docs/tools/h
...
+You can add additional functionality through Add-Ons which are just preconfigured docker images to run with Home Assistant.
```
Comment (frenck) [inline source/_docs/installation.markdown]:
```suggestion
You can add additional functionality through add-ons which are just preconfigured docker images to run with Home Assistant.
```

File: source/_docs/installation.markdown [hunk 10]
```diff
@@ -131,3 +131,124 @@ For more details about `hass`, please refer to the [tools section](/docs/tools/h
...
+
+You will get:
+- You can use Add-Ons from the core team
```
Comment (frenck) [inline source/_docs/installation.markdown]:
```suggestion
- Access to the add-ons from the core team.
```

File: source/_docs/installation.markdown [hunk 11]
```diff
@@ -131,3 +131,124 @@ For more details about `hass`, please refer to the [tools section](/docs/tools/h
...
+- You can use Add-Ons from the community
```
Comment (frenck) [inline source/_docs/installation.markdown]:
```suggestion
- Access to the add-ons from the community.
```

File: source/_docs/installation.markdown [hunk 12]
```diff
@@ -131,3 +131,124 @@ For more details about `hass`, please refer to the [tools section](/docs/tools/h
...
+- You can use any other Add-Ons
```
Comment (frenck) [inline source/_docs/installation.markdown]: Considering the above 2...  what are the "other" add-ons?

File: source/_docs/installation.markdown [hunk 13]
```diff
@@ -131,3 +131,124 @@ For more details about `hass`, please refer to the [tools section](/docs/tools/h
...
+- You can build your own Add-Ons (building and configuring docker images)
```
Comment (frenck) [inline source/_docs/installation.markdown]:
```suggestion
- To possibility to build your own add-ons.
```

File: source/_docs/installation.markdown [hunk 14]
```diff
@@ -131,3 +131,124 @@ For more details about `hass`, please refer to the [tools section](/docs/tools/h
...
+- HassOS will run on the underling hardware
```
Comment (frenck) [inline source/_docs/installation.markdown]:
```suggestion
- HassOS will run on the underlying hardware.
```

File: source/_docs/installation.markdown [hunk 15]
```diff
@@ -131,3 +131,124 @@ For more details about `hass`, please refer to the [tools section](/docs/tools/h
...
+- You do/can not need to touch the operating system
```
Comment (frenck) [inline source/_docs/installation.markdown]:
```suggestion
- Automatic updates of the operating system (OTA).
```

File: source/_docs/installation.markdown [hunk 16]
```diff
@@ -131,3 +131,124 @@ For more details about `hass`, please refer to the [tools section](/docs/tools/h
...
+- You can update HassOS via Hass.io GUI
```
Comment (frenck) [inline source/_docs/installation.markdown]:
```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]
Left a partial review, which contains a trend you can apply to the rest of the PR.

Secondly, I agree with people above, this needs a separate page.

**Comment by ajfriesen** [comment]
Will have a look later in the day.

File: source/_docs/installation.markdown (hunk 1 above)
Comment (ajfriesen) [reply source/_docs/installation.markdown]: Each installation method will give you home assistant with all features.

Often people do not understand that the hassio add-ons are not home assistant functionality itself, it is just more or less preconfigured software to work with home assistant.

File: source/_docs/installation.markdown (hunk 3 above)
Comment (ajfriesen) [reply source/_docs/installation.markdown]: I just added this comment as I saw this: 

```
When you use this installation method, the core SSH add-on may not function correctly. If that happens, use the community SSH add-on. Some of the documentation might not work for your installation either.
```
[Source](https://www.home-assistant.io/hassio/installation/#alternative-install-on-generic-linux-server)

I do not know how deep the add-ons are depending on hassio and therefore on hassOS.

If there is no other dependency than the architecture (for example there is only an arm6 container available but not for arm7) then we should not mention that.

**Comment by ajfriesen** [comment]
I totally agree to use a dedicated page and did just that.
Any ideas where it should be linked?

Sorry I got somehow confused with the branching and forking.
First time working on a community project on github.
Hope that I will learn more about the process :-)

**Pull request closed without merging**